.github/workflows/gradle.version.yaml

```diff
@@ -46,6 +46,16 @@ jobs:
           fetch-depth: 1000
           fetch-tags: true
 
+      - id: getVersionOrdering
+        name: "Get version ordering"
+        run: |
+          uses_mc_version_order_first=$(cat gradle.properties | grep usesMCVersionOrderFirst | cut -d "=" -f 2)
+          if [ -z "$uses_mc_version_order_first" ]; then
+            uses_mc_version_order_first=false
+          fi
+          echo "Using MC version order first: $uses_mc_version_order_first"
+          echo "uses_mc_version_order_first=$uses_mc_version_order_first" >> "$GITHUB_OUTPUT"
+
```

Comment:
This is not needed in the NG7 Branch. 
Anything that uses this branch should use <modversion>-<mcversion>